php/README.markdown

```diff
@@ -0,0 +1,35 @@
+# DNSimple API PHP Examples
+
+The scripts in this directory demonstrate how to use the DNSimple PHP API wrapper to connect to the DNSimple API.
+
+## Running
+
+First you will need to ensure PHP is installed and then run `php -v`.
+
+Once you install the dependencies, create a file called `token.php` and put the following into it:
```

Comment:
We should explain how to install the dependencies. I suggest we add a basic composer.json file at `/php` with `dnsimple/dnsimple` as the only dependency and then expand these instructions with something that explains how to get `composer.phar` and run `php composer.phar install`

Reply:
That's a great idea! 👍

Reply:
Quick question though:

Don't you think that adding the composer.json file would be enough? Maybe stating in the readme that they could use composer to install the dependencies... I don't necessarily want to go through the lengths of explaining how to install composer, etc...